test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
That's the thing, I have absolutely no idea.  The two implementations are more or less the same and should functionally operate the same but they don't, I think it is due to the "before" calls in the `.execute` but can't prove / fix it